assets/css/pages/blog-landing/index.css

```
@@ -53,7 +53,7 @@ section.heading {
 }
 
 .footer {
-    background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../images/spacebg.jpg");
+    background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../../../images/spacebg.jpg");
```

After the issues we are having with the cookies.js, I think we should make these links with absolute paths.
```suggestion
    background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("/assets/images/spacebg.jpg");
```
Not sure if that's correct, fyi